src/pages/billing/Project.js

```diff
@@ -487,6 +488,7 @@ const ProjectDetail = ({ billingProject, reloadBillingProject, billingAccounts,
         style: { marginTop: '2rem', textTransform: 'none', padding: '0 1rem', height: '1.5rem' },
         tabStyle: { borderBottomWidth: 4 },
         value: tab,
+        key: `tabBarKey${isAlphaSpendReportUser}`, // The Spend report tab is only present for alpha users, so force recreation.
```

Comment:
Sure. The issue is that the array of tabs within `SimpleTabBar` cannot change on re-renders. Thus we have to force React to create a new instance of `SimpleTabBar` when the value of `isAlphaSpendReportUser` changes, since we only show the "Spend report" tab when that value is true (and we don't hear back from the isMember Ajax call until after the UI is initially rendered).

If that still isn't making sense, this [Slack thread](https://broadinstitute.slack.com/archives/C01EHNUM73R/p1644873326437099) might help.